Makefile.am

```diff
@@ -17,6 +19,27 @@ DISTCLEANFILES =
 CLEANFILES =
 MOSTLYCLEANFILES =
 noinst_PROGRAMS =
+MAINTAINERCLEANFILES = \
+    $(GITIGNORE_MAINTAINERCLEANFILES_TOPLEVEL) \
+    $(GITIGNORE_MAINTAINERCLEANFILES_MAKEFILE_IN) \
+    $(GITIGNORE_MAINTAINERCLEANFILES_M4_LIBTOOL) \
+    src_vars.mk \
+    Makefile-fuzz-generated.am \
+    aminclude_static.am \
+    m4/ax_ac_append_to_file.m4 \
+    m4/ax_ac_print_to_file.m4 \
+    m4/ax_add_am_macro_static.m4 \
+    m4/ax_am_macros_static.m4 \
+    m4/ax_check_compile_flag.m4 \
```

Comment:
On my system, the otherwise untracked `m4/ax_check_gnu_make.m4` needs to be added to `MAINTAINERCLEANFILES`.

Reply:
I'll add that as well...